Sources/FirebaseCore/FirebaseOptions+Swift.swift

```diff
@@ -138,18 +138,19 @@ extension FirebaseOptions {
   /// - Parameters:
   ///   - path: The path where the config file can be found.
   ///   - format: A format which describes how the content of the file should be treated.
-  public init?(_contentsOfFile path: String, format: ConfigFormat) {
-    switch format {
-      case .json:
-        do {
-          let config = try String(contentsOfFile: path)
+  public init?(_contentsOfFile path: URL, format: ConfigFormat) {
+    do {
+      let data = try Data(contentsOf: path)
```

Comment:
I think that if we do:

```swift
guard let data = try? Data(contentsOf: path) else { return nil }
```

we can avoid indentation and the large `do { } catch { }` block, particularly as we drop the error on the floor anyway.

Reply:
Fixed